src/shared/components/LomFieldsInput/LomFieldsInput.helpers.ts

```diff
@@ -19,9 +20,15 @@ export const mapOptionsToLomFields = (
 	});
 };
 
-export const getParentContext = (loms: Avo.Lom.LomField[], allLoms: Avo.Lom.LomField[]) => {
+export const getParentContext = (
+	loms: Avo.Lom.LomField[],
+	allLoms: Avo.Lom.LomField[]
+): LomFieldSchema[] => {
 	return compact(
 		loms.map((lom) => {
+			if (isNil(lom.broader)) {
+				return;
```

Comment:
return: means you're done with executing a function that has no return value
return null; means you return an empty value that you then filter out using the compact statement
```suggestion
				return null;
```